packages/core/phase1/triggers/generate.ts

```diff
@@ -15,11 +17,22 @@ const independentTriggerFn = (aho: AnnotatedHearingOutcome) =>
 
 const generateSetOfTriggers = (
   generator: TriggerGenerator,
-  aho: AnnotatedHearingOutcome,
-  existingTriggers: Trigger[] = []
+  aho: AnnotatedHearingOutcome | PncUpdateDataset,
+  existingTriggers: Trigger[] = [],
+  generatePostUpdateTriggers = false
 ): Trigger[] => {
-  const generatedTriggers = generator(aho, { triggers: existingTriggers })
-  const filteredTriggers = filterExcludedTriggers(aho, generatedTriggers)
+  const generatedPreUpdateTriggers = generator(aho, { triggers: existingTriggers })
+  const filteredPreUpdateTriggers = filterExcludedTriggers(aho, generatedPreUpdateTriggers)
+
+  let generatedPostUpdateTriggers: Trigger[] = []
+  let filteredPostUpdateTriggers: Trigger[] = []
+  if (generatePostUpdateTriggers && "PncOperations" in aho) {
+    generatedPostUpdateTriggers = identifyPostUpdateTriggers(aho)
```

Comment:
This is temporary until we move the post-update triggers logic to separate trigger files. Once it's done, generator function will generate both pre and post update triggers.